test/spec/smartbanner_spec.js

```diff
@@ -99,13 +182,16 @@ describe('SmartBanner', function() {
     context('without options', function() {
 
       before(function() {
-        global.document = jsdom.jsdom('<html></html>', { userAgent: USER_AGENT_IPHONE_IOS9 });
+        global.document = jsdom.jsdom('<html></html>', {
```

Comment:
Let's revert this, again, not to generate a diff in places where your feature is not relevant and should not touch down.